### PR TITLE
Update model path for microsoft-phi-2

### PR DESCRIPTION
**Pull request by choudharykartik**
pipeline run link: https://dev.azure.com/msdata/Vienna/_build/results?buildId=119865522&view=results
inference results:
```
[
  {
    "endpoint_name": "msft-phi-2-NC6s-v3-1709103469",
    "deployment_name": "default",
    "inference_payload": {
      "input_data": {
        "input_string": [
          "My name is John and I am",
          "Once upon a time,"
        ],
        "parameters": {
          "max_new_tokens": 25,
          "do_sample": true,
          "temperature": 0.5,
          "top_p": 0.5
        }
      }
    },
    "response": "[{\"0\": \"My name is John and I am a student at XYZ High School. I am writing to you to request an extension for my history assignment that is due on\"}, {\"0\": \"Once upon a time, in a small town called Willowbrook, there lived a young woman named Alice. Alice was known for her exceptional intelligence and her\"}]",
    "sku": "Standard_NC6s_v3",
    "status": "Completed"
  },
  {
    "endpoint_name": "msft-phi-2-F8s-v2-1709107505",
    "deployment_name": "default",
    "inference_payload": {
      "input_data": {
        "input_string": [
          "My name is John and I am",
          "Once upon a time,"
        ],
        "parameters": {
          "max_new_tokens": 25,
          "do_sample": true,
          "temperature": 0.5,
          "top_p": 0.5
        }
      }
    },
    "response": "[{\"0\": \"My name is John and I am a student at XYZ High School. I am writing to you to request an extension for my history project, which is due\"}, {\"0\": \"Once upon a time, in a small town called Willowbrook, there lived a young woman named Alice. Alice was known for her exceptional intelligence and her\"}]",
    "sku": "Standard_F8s_v2",
    "status": "Completed"
  },
  {
    "endpoint_name": "msft-phi-2-DS3-v2-1709111491",
    "deployment_name": "default",
    "inference_payload": {
      "input_data": {
        "input_string": [
          "My name is John and I am",
          "Once upon a time,"
        ],
        "parameters": {
          "max_new_tokens": 25,
          "do_sample": true,
          "temperature": 0.5,
          "top_p": 0.5
        }
      }
    },
    "response": "[{\"0\": \"My name is John and I am a student at XYZ Elementary School. I am writing to you to ask for your permission to go on a field trip to\"}, {\"0\": \"Once upon a time, in a small town called Oakville, there lived a young girl named Lily. Lily was a curious and adventurous girl who loved\"}]",
    "sku": "Standard_DS3_v2",
    "status": "Completed"
  },
  {
    "endpoint_name": "msft-phi-2-L8s-v2-1709103462",
    "deployment_name": "default",
    "inference_payload": {
      "input_data": {
        "input_string": [
          "My name is John and I am",
          "Once upon a time,"
        ],
        "parameters": {
          "max_new_tokens": 25,
          "do_sample": true,
          "temperature": 0.5,
          "top_p": 0.5
        }
      }
    },
    "response": "[{\"0\": \"My name is John and I am a student at XYZ High School. I am writing to you to request an extension for my history assignment that is due on\"}, {\"0\": \"Once upon a time, in a small town called Willowbrook, there lived a young woman named Alice. Alice was known for her exceptional intelligence and her\"}]",
    "sku": "Standard_L8s_v2",
    "status": "Completed"
  },
  {
    "endpoint_name": "msft-phi-2-E2s-v3-1709103474",
    "deployment_name": "default",
    "inference_payload": {
      "input_data": {
        "input_string": [
          "My name is John and I am",
          "Once upon a time,"
        ],
        "parameters": {
          "max_new_tokens": 25,
          "do_sample": true,
          "temperature": 0.5,
          "top_p": 0.5
        }
      }
    },
    "response": "[{\"0\": \"My name is John and I am a student at XYZ High School. I am writing to you to request an extension for my history project, which is due\"}, {\"0\": \"Once upon a time, in a small town called Oakville, there lived a young girl named Lily. Lily was a curious and adventurous girl who loved\"}]",
    "sku": "Standard_E2s_v3",
    "status": "Completed"
  }
]
```

### CODE REVIEW

**Comment by novaturient95** [comment]
Please add validation results in description

File: assets/models/system/microsoft-phi-2/model.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 path:
   container_name: models
-  container_path: huggingface/microsoft-phi-2/1704358140/mlflow_model_folder
+  container_path: huggingface/microsoft-phi-2/1709113941/mlflow_model_folder
```
Comment (novaturient95) [inline assets/models/system/microsoft-phi-2/model.yaml:3]: Please add validation results

File: assets/models/system/microsoft-phi-2/model.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
   storage_name: automlcesdkdataresources
   type: azureblob
```
Comment (novaturient95) [inline assets/models/system/microsoft-phi-2/model.yaml:5]: SHA in model spec? 
version change

**Review by novaturient95 — CHANGES_REQUESTED** [review]

File: assets/models/system/microsoft-phi-2/model.yaml (hunk 2 above)
Comment (choudharykartik) [reply assets/models/system/microsoft-phi-2/model.yaml:5]: added sha in spec.yaml
version is already changed in last PR that version is not released yet

**Review by novaturient95 — APPROVED** [review]

**Pull request merged**